docs/settings/security-settings.asciidoc

```diff
@@ -362,52 +358,106 @@ To enable the <<xpack-security-ecs-audit-logging, ECS audit logger>>, specify wh
 [source,yaml]
 ----------------------------------------
 xpack.security.audit.appender:
-  kind: file
-  path: /path/to/audit.log
+  kind: rolling-file
+  path: ./audit.log
+  policy:
+    kind: time-interval
+    interval: 24h <1>
+  strategy:
+    kind: numeric
+    max: 10 <2>
   layout:
     kind: json
 ----------------------------------------
+<1> Rotates log files every 24 hours
+<2> Keeps maximum of 10 log files before deleting older ones
 
 | `xpack.security.audit.appender.kind`
-| Required. Specifies where audit logs should be written to. Allowed values are `console` or `file`.
-|===
+| Required. Specifies where audit logs should be written to. Allowed values are `console`, `file` or `rolling-file`. 
+
+Refer to <<audit-logging-file-appender, file appender>> and <<audit-logging-rolling-file-appender, rolling file appender>> for appender specific settings.
+
+| `xpack.security.audit.appender.layout.kind`
+| Required. Specifies how audit logs should be formatted. Allowed values are `json` or `pattern`.
```

Comment:
Do you think it's worth calling out that `json` is the recommended format, and the only format supported by our filebeat module?